google-cloud-bigquery/pom.xml

```diff
@@ -99,12 +99,9 @@
       <artifactId>truth</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.easymock</groupId>
-      <artifactId>easymock</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.objenesis</groupId>
-      <artifactId>objenesis</artifactId>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
Let's use mokito-core (which is alr in shared-deps) and get rid of Hamcrest as well #357